src/NuGet.Core/NuGet.Protocol/LegacyFeed/ODataServiceDocumentUtils.cs

```diff
@@ -3,7 +3,9 @@
 
 using System;
 using System.Globalization;
+using System.Net;
 using System.Net.Http;
+using System.Net.Sockets;
```

Comment:
Remove both of these additions.  They're unused.

Reply:
fixed in next iteration.